docs/src/api/class-consolemessage.md

```diff
@@ -5,6 +5,22 @@
 ## method: ConsoleMessage.args
 - returns: <[Array]<[JSHandle]>>
 
+List or arguments passed to a `console` function call.
+
+```js
+const { chromium } = require('playwright');  // Or 'firefox' or 'webkit'.
+
+(async () => {
+  const browser = await chromium.launch();
+  const page = await browser.newPage();
+  page.evaluate(() => console.log('hello', 5, {foo: 'bar'})),
```

Comment:
this is currently invalid JavaScript syntax. Probably you want waitForEvent("console") etc.

Reply:
Good catch @mxschmitt. How about now?